tests/clv/models/test_pareto_nbd.py

```diff
@@ -132,12 +132,20 @@ def test_model_convergence(self, fit_method, rtol):
             s_prior=s_prior,
             beta_prior=beta_prior,
         )
-        sample_kwargs = (
-            dict(random_seed=self.rng, chains=2, step=pm.Slice())
-            if fit_method == "mcmc"
-            else {}
-        )
-        model.fit(fit_method=fit_method, progressbar=False, **sample_kwargs)
+
+        if fit_method == "mcmc":
+            with model.model:
+                model.fit(
+                    fit_method=fit_method,
+                    random_seed=self.rng,
+                    chains=2,
+                    step=pm.Slice(),
```

Comment:
Can we not use NUTS?

Reply:
Needs https://github.com/pymc-devs/pytensor/pull/283 to be usable at all with a simple 2k rows dataset

Reply:
I've marked this model as experimental because NUTS is having convergence errors due to `pytensor.Tensor.hyp2f1` in the model `logp`, but @ricardoV94 has been trying out some improvements for that function.